argo/values.schema.json

```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "templateName": {
+      "description": "Name to assign to the generated WorkflowTemplate object",
+      "type": "string"
+    }
+  },
+  "required": ["name"]
```

Comment:
should this be templateName?

Reply:
Yes, good catch!

Reply:
If only we had a schema for our schema...